src-docs/src/views/guidelines/getting_started/getting_started.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+
+import { EuiCodeBlock, EuiCode, EuiText, EuiSpacer } from '../../../../../src';
+
+import { AppSetup } from './_app_setup';
+import { Tokens } from './_tokens';
+import { CustomizeTokens } from './_customize_tokens';
+
+export const GettingStarted = {
+  title: 'Getting started',
+  sections: [
+    {
+      title: 'Installation',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              EUI is published through{' '}
+              <a href="https://www.npmjs.com/package/@elastic/eui">NPM</a> as a
+              dependency. To install EUI into an existing project, use the{' '}
+              <EuiCode>yarn</EuiCode> CLI (<EuiCode>npm</EuiCode> is not
+              supported).
+            </p>
+          </EuiText>
+          <EuiSpacer />
+          <EuiCodeBlock language="bash" isCopyable fontSize="m">
+            {'yarn add @elastic/eui'}
+          </EuiCodeBlock>
+          <EuiSpacer />
+          <EuiText grow={false}>
+            <p>
+              Note that EUI has{' '}
+              <a href="https://github.com/elastic/eui/package.json">
+                several <EuiCode>peerDependencies</EuiCode> requirements
+              </a>{' '}
+              that will also need to be installed if starting with a blank
+              project. You can read more about other ways to{' '}
+              <a href="https://github.com/elastic/eui/blob/master/wiki/consuming.md">
+                consume EUI
+              </a>
+              .
+            </p>
+          </EuiText>
+        </>
+      ),
+    },
+    {
+      title: 'Setting up your application',
+      wrapText: false,
+      text: <AppSetup />,
+    },
+    {
+      title: 'Styling your application',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              To build your custom components using EUI theme variables,
+              functions, and mixins, you will need to consume them through one
+              of the <a href="/#/theming/sass">Sass</a>,{' '}
+              <a href="/#/theming/theme-provider">Emotion</a>, or{' '}
```

Comment:
It's better if we use the Link from `react-router-dom`:

```suggestion
              of the <Link to="/theming/sass">Sass</Link>,{' '}
              <Link to="/theming/theme-provider">Emotion</Link>, or{' '}
```